.github/CONTRIBUTING.md

```diff
@@ -49,6 +67,9 @@ outlined on that page and do not file a public issue.
 * 4 spaces for indentation rather than tabs
 * Public classes and methods must contain header documentation
 * Use plain C functions whenever possible (as opposed to class methods)
+* Restrict subclassing (objc macro, final in examples)
```

Comment:
Nevermind I figured out the 2nd part of my question just need a little help with the macro question and I'm good to go! 💯 @rnystrom